extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardModel.ts

```diff
@@ -50,9 +50,27 @@ export class BdcDashboardModel {
 			}),
 			getEndPoints(this.url, this.username, this.password, true).then(response => {
 				this._endpoints = response.endPoints || [];
+				fixEndpoints(this._endpoints);
 				this._endpointsLastUpdated = new Date();
 				this._onDidUpdateEndpoints.fire(this.serviceEndpoints);
 			})
 		]).catch(error => showErrorMessage(error));
 	}
 }
+
+/**
+ * Applies fixes to the endpoints received so they are displayed correctly
+ * @param endpoints The endpoints received to modify
+ */
+function fixEndpoints(endpoints: EndpointModel[]) {
+	endpoints.forEach(e => {
+		if (e.name === Endpoint.metricsui && e.endpoint && e.endpoint.indexOf('/d/wZx3OUdmz') === -1) {
+			// Update to have correct URL
+			e.endpoint += '/d/wZx3OUdmz';
```

Comment:
is this temporary?

Reply:
Ideally. The platform team is aware of the issue but I don't believe there's a fix planned anytime soon. I added the extra logic to try and detect if it ever is fixed though so we don't break immediately if they do.